docs/logging-usage.rst

```diff
@@ -205,13 +205,74 @@ Delete a metric:
    >>> metric.exists()  # API call
    False
 
-
 Export log entries using sinks
 ------------------------------
 
 Sinks allow exporting entries which match a given filter to Cloud Storage
 buckets, BigQuery datasets, or Cloud Pub/Sub topics.
 
+Export to Cloud storage
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure that the storage bucket you want to export logs too has
+`cloud-logs@google.com` as the owner. See `Set permission for writing exported logs`_.
```

Comment:
If you want `<code>` in an RST file it needs to have two backticks on either side